app/app.js

```diff
@@ -13,8 +13,8 @@ process.on('uncaughtException', logErrorHandler);
 
 app.use(cors());
 app.use(express.json({ extended: true }));
-app.use(authMiddleware);
 app.use('/', router);
+app.use(authMiddleware);
```

Comment:
you have public route: '/login'
all other routes are private and should be validated by authMiddleware
you can do like this
```javascript
  app.use(publicRoutes);
  app.use(authMiddleware);
  app.use(priveteRoutes);
```
publicRoutes has login routes
all other routes are included into privateRoutes

Reply:
done